gst.so/src/gst_worker.cpp

```diff
@@ -598,12 +609,8 @@ uint64_t GSTWorker::time_diff(
 void GSTWorker::log_to_json(const std::string &key, const std::string &value,
```

Comment:
Can we move this function to rvs util or separate json util file ?

Reply:
the primary fields usually vary between modules and hence i have put it here. Once other modules are refactored we ca take larger subset of necessary ones and move them to util